deploy/cr.yaml

```diff
@@ -583,6 +583,18 @@ spec:
 #    proxysqlParams: "--custom-labels=CUSTOM-LABELS"
 #    containerSecurityContext:
 #      privileged: false
+#    readinessProbes:
```

Comment:
Please add these options to helm chart as well

Reply:
https://github.com/percona/percona-helm-charts/pull/472